jaxdecomp/_src/fft.py

```diff
@@ -42,6 +42,7 @@ def _str_to_fft_type(s: str) -> xla_client.FftType:
 #@partial(jit, static_argnums=(1, 2, 3, 4))
 def sfft(x,
          fft_type: Union[xla_client.FftType, str],
+         yz_slab=False,
```

Comment:
Consistency should be enforced

Here `yz_slab` after `fft_type`, before `adjoint`